taxi-service (#1)
SerhiiPankov/taxi: README.md

# Taxi manager
-store data for taxi management
-administration the driver-car relationship
-provide stored information for registered drivers
# Features
- store information about the model, manufacturer

> - and a list of taxi drivers who can use this car

Reviewer: remove these lines

> - authentication system for drivers
> - registration system for drivers
> - add manufacturer, car,
> - associate the car with the driver.

Reviewer: ```suggestion
- register a new driver
- log in, log out
- add manufacturer, car,
- add a new driver to the car
```

> - display a general list of drivers with the ability to delete
> - display a general list of manufacturers with the ability to delete
> - display a general list of cars
> - display a list of cars of an authenticated driver

Reviewer: ```suggestion
- display al list of drivers with the ability to delete
- display a list of manufacturers with the ability to delete
- display a list of cars
- display a list of cars of an authenticated driver
```

# Used technologies
- Java Development Kit (JDK)
- Java Servlet API
- Java Database Connectivity (JDBC)
- Java Server Pages (JSP)
- JSP Standard Tag Library (JSTL)

> - MySql database

Reviewer: ```suggestion
- MySQL
```

> - WAR packaging

Reviewer: remove this line

- Maven
- Tomcat
# To start a project you need
- add Tomcat server configuration 
- MySql (or another) DB
- create DB using file init_db.sql

> - enter url, login, password, driver for your Db to ConnectionUtil.class

Reviewer: 1. Fill (real variables names in ConnectionUtil) to ConnectionUtil.
2. Initialize DB using init_db.sql file.
3. Configure Tomcat
4. Launch the project
